Name: Option In Context Menu
Version: 1.3.5-1
Author: Fritos1406
Description: Gives the option of "Open URL" in the context menu

--- .orig/usr/palm/applications/com.palm.app.browser/app/controllers/page-assistant.js
+++ /usr/palm/applications/com.palm.app.browser/app/controllers/page-assistant.js
@@ -611,6 +611,7 @@
 		if (event.altKey) {
 
 			var popupItems = [
+			    {label: $L('Open URL'), command:'openUrl'},
 				{label: $L('Open In New Card'), command:'openNew'},
 				{label: $L('Share Link'), command:'shareUrl'},
 				{label: $L('Copy URL'), command:'copyUrl'},
@@ -656,6 +657,9 @@
 			var popupSelectFunc = function(value) {
 				selectedCommand = value;
 				switch (value) {
+					case 'openUrl':
+						this.openUrl(urlInfo.url);
+						break;				
 					case 'openNew':
 						this._newBrowserPage(urlInfo.url);
 						break;
@@ -682,6 +686,7 @@
 				imageInfo = response;
 				var usedItems = [];
 				if (urlInfo.url) {
+				    usedItems.push( findItem('openUrl') );
 					usedItems.push( findItem('openNew') );
 					usedItems.push( findItem('shareUrl') );
 					usedItems.push( findItem('copyUrl') );
